schema/person-statement.json

```diff
@@ -182,6 +167,32 @@
     },
     "replacesStatements": {
       "$ref": "components.json#/definitions/ReplacesStatements"
+    },
+    "pepStatus": {
+      "type": "object",
+      "title": "Politically Exposed Person status",
+      "description": "Information about whether, and how, the person described by this statement is politically exposed. Use this property only if PEP declarations are expected as part of beneficial ownership declarations.",
+      "required": [
+        "hasPepStatus"
+      ],
+      "properties": {
+        "hasPepStatus": {
```

Comment:
@kindly - do go ahead and adjust this to a string/enum as you suggested in conversation, so that we have an 'unknown' value. I think we prob have to massage the naming too, so that we don't end up with a pseudo-boolean field with confusing "true"/"false" string values. How about 'personsPepStatus' with values 'isPep', 'isNotPep' and 'unknown'. I'll suggest a rewording of the description below.

Reply:
After a chat with @kindly the plan is to rename some properties to make this align more neatly.